source/SIL.AppBuilder.Portal.Frontend/src/data/store.ts

```diff
@@ -29,6 +29,7 @@ class CustomJSONAPISerializer extends JSONAPISerializer {
   resourceKey(type: string) { return 'remoteId'; }
 }
 
+export const serializer = new JSONAPISerializer({ schema, keyMap });
```

Comment:
in case we ever need to normalize data outside of orbit, we can do `serializer.deserializeDocument(...)`